tedana/resources/decision_trees/kundu.json

```
@@ -98,7 +109,9 @@
                 "left2": "variance explained",
                 "op2": ">",
                 "right2": "median_varex",
-                "tag_if_true": "Unlikely BOLD"
+                "tag_if_true": "Unlikely BOLD",
+                "log_extra_info": "",
+                "log_extra_report": "Any components with higher S0 model beta map-F-statistic map Dice similarity index than T2 model beta map-F-statistic map Dice similarity index and greater than median variance explained are rejected."
```

I don't understand the wording here: `beta map-F-statistic map`.

---

What about this?

```suggestion
                "log_extra_report": "Any components with (1) greater than median variance explained and (2) greater correspondence between the thresholded and binarized S0-model beta map and the thresholded and binarized F-statistic loading map than between the thresholded and binarized T2-model beta map and the thresholded and binarized F-statistic loading map (as quantified with the Dice similarity index) are rejected."
```

---

We have a lot of repeated metrics and it doesn't make sense to re-define them every time they are used. That said, I'm going through our docs and the only place we really define them is in the API ( https://tedana.readthedocs.io/en/stable/generated/tedana.metrics.dependence.html#module-tedana.metrics.dependence) and some of those definitions could use more clarity. 

I thought we had a descriptive list of all the metrics outputted in the component table, but maybe not? For now, maybe we can link to the metric descriptors in the API in a `_comment` for the whole json file (and maybe update particularly confusing descriptions int he API). That would free us to use more conceptual descriptions in the comments. For example:

```suggestion
                "log_extra_report": "For components with more than median variance, if the peak clusters for the S0-model overlap more with the peaks of the ICA component's weighting map than the peak clusters for the T2-model overlaps with the ICA weighting map, then reject.
```
(That might be better as a `_comment` that in the report)

---

I re-found the descriptive list of metrics. It's included in the outputs in the `desc-tedana.metrics.json` file. This makes a lot of sense since the specific metrics might vary depending on pipeline. That said, the descriptions could be improved. I could see explicitly referencing that file in this json (as part of this PR) and improving those descriptions either in this PR or in a separate PR.